client/package.json

```diff
@@ -22,6 +22,7 @@
     "fscreen": "^1.2.0",
     "http-proxy-middleware": "^2.0.1",
     "react": "^17.0.2",
+    "react-confetti": "^6.1.0",
```

Comment:
The best way to solve conflict errors with package-lock.json  pull the changes from upstream, then install your dependencies on the file pacakge.json file